sdk/src/main/java/com/sygic/travel/sdk/auth/facade/AuthFacade.kt

```diff
@@ -0,0 +1,170 @@
+package com.sygic.travel.sdk.auth.facade
+
+import com.sygic.travel.sdk.Callback
+import com.sygic.travel.sdk.auth.model.AuthorizationRequest
+import com.sygic.travel.sdk.auth.model.UserRegistrationRequest
+import com.sygic.travel.sdk.auth.model.UserRegistrationResponse
+import com.sygic.travel.sdk.auth.model.UserSession
+import com.sygic.travel.sdk.auth.service.AuthService
+import com.sygic.travel.sdk.auth.service.AuthStorageService
+import com.sygic.travel.sdk.utils.runAsync
+import com.sygic.travel.sdk.utils.runWithCallback
+
+
+class AuthFacade(
+	private val authService: AuthService,
+	private val authStorageService: AuthStorageService
+) {
+
+	companion object {
+		const val CLIENT_ID = "sygictravel_android_sdk_demo"
+		const val DEVICE_PLATFORM = "android"
+	}
+
+	fun loginUserByPassword(
```

Comment:
I've just found out that `loginUserByPassword` is not language correct :( So it should be `loginUserWithPassword` same for other methods... Nobody noticed in spec. Sorry for that...

Reply:
:heavy_check_mark: